### PR TITLE
SetFit: When logging probs, log them not as logits

### PR DESCRIPTION
**Pull request by franz101**
This is a proposed fixed that might resolve quirks around the DEP reported for setfit

### CODE REVIEW

**Review by setu4993 — APPROVED** [review]

**Comment by Ben-Epstein** [comment]
@franz101 please bump the version on init so we can release this

**Comment by franz101** [comment]
@Ben-Epstein done

**Review by Ben-Epstein — APPROVED** [review]
